components/global/global.js

```diff
@@ -1,5 +1,8 @@
 import React, { Component, PropTypes } from 'react';
-import { render, unmountComponentAtNode } from 'react-dom';
+import {
+  unstable_renderSubtreeIntoContainer, // eslint-disable-line
```

Comment:
because, in case, when we will use `frint`, `redux`, or something like that, we have `Provider`, which provide `store` or another stuff, it donesn't work as expected.

because `render` will render content in to `body`, this content has no access to `Provider`, and component which connected to store.

`unstable_renderSubtreeIntoContainer` will work like `pipeline`, and we can render content into `target`. 
something like that ;)